vmdb/product/toolbars/ems_infra_center_tb.yaml

```diff
@@ -22,6 +22,11 @@
       :text: 'Edit this #{ui_lookup(:table=>"ems_infra")}'
       :title: 'Edit this #{ui_lookup(:table=>"ems_infra")}'
       :url: '/edit'
+    - :button: ems_infra_scale
+      :image: edit
+      :text: 'Scale this #{ui_lookup(:table=>"ems_infra")}'
+      :title: 'Scale this #{ui_lookup(:table=>"ems_infra")}'
```

Comment:
seems like something changed here

it should look like 
'Edit this <%= ui_lookup(:table=>"ems_infra") %>'

or at least I see that in current master, in UI now I see the code

@martinpovolny was this #{ui_lookup syntax a bug?